packages/l10n/src/time-zone-information.ts

```diff
@@ -50,12 +88,21 @@ export const timeZonesShape = PropTypes.objectOf(
  * If running through webpack, code splitting makes `getTimeZonesForLocale`
  * a function that asynchronously loads the country data.
  */
-const getTimeZonesForLocale = async (locale: string) => {
+const getTimeZonesForLocale = async (locale: string): Promise<TimeZones> => {
   const supportedLocale = getSupportedLocale(locale);
   // Use default webpackMode (lazy) so that we generate one file per locale.
   // The files are named like "time-zone-data-en-json.chunk.js" after compilation
   // https://webpack.js.org/api/module-methods/#import-
-  const timeZones = await getImportChunk(supportedLocale);
+  let localeTranslations = await getImportChunk(supportedLocale);
+  // if there are no translations for the desired locale, fall back to en/default
+  if (
+    !localeTranslations.default ||
+    Object.keys(localeTranslations.default).length === 0
+  ) {
+    localeTranslations = await getImportChunk('en');
+  }
+  // create time zone object with abbreviations and offsets
```

Comment:
Don't we have to handle this on a per timezone basis also and not on a chunk basis? What if we also add the new timezone name (EN version) to any `<locale.>json` file if the timezone is new. That would have it default to EN until we pull in something else from transifex.

Reply:
This works for me if it works for you

Reply:
Otherwise I don't see how we can show anything if we have say 22 translations in en.json and 20 in de.json (as two are new). I don't see how de.json being loaded then (as it's not empty) would then mean that it can never show anything for the two new TZs if I am not mistaken.

Reply:
I didn't know whether it'd be better to wait for a translation before making a 'new' timezone available in the final tz list, or to add the tz to the locales immediately.

We can talk more about the workflow when we meet about transifex, but I think there are pros and cons to each approach.  If we decide to add the translation using the CLI tool, I will refactor it so that it writes to all locale files as well as `core.json`.

Reply:
> I didn't know whether it'd be better to wait for a translation before making a 'new' timezone available in a locale's tz list, or to add the tz to the locale files immediately.

I think that's a 🐔 🥚 problem if I understand your concern right. Let's talk tomorrow.